recipe/meta.yaml

```diff
@@ -38,7 +38,7 @@ requirements:
     - {{ compiler('cxx') }}  # [not win]
     - {{ compiler('m2w64_cxx') }}  # [win]
     - python
-    - {{ pin_compatible('numpy') }}
```

Comment:
The pinning statement should stay. Why did this get removed?

Reply:
If you need a min version of numpy, you should add a constraint in the host section like this `numpy 1.15.*` or whatever. See the docs here https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
it build against 1.11. Is that normal with conda-forge?

Reply:
Yes. The build is against an older version so that it will run against any newer version past that.

Reply:
Ok, I can rewind it back

Reply:
You should always build against the oldest version possible down to 1.11.*. However, if the package can only run on say 1.14.* and later, then by following the instructions in the link above, the `pin_compatible` statement will automatically put in the right dependency range in the run section.